src/python/espressomd/particle_data.pyx

```diff
@@ -1678,6 +1678,61 @@ cdef class ParticleList:
                     if (p.type == t or t == "all"):
                         vtk.write("{} {} {}\n".format(*p.v))
 
+
+    def writevsf(self, fp, types='all'):
+        """
+        writes a VST (VTF Structure Format) to a file.
```

Comment:
Can you please also add a documentation in the io part of the sphinx documentation and shortly describe what your function does?